### PR TITLE
[docs] Remove broken links

### LINKED ISSUE #140: Remove broken link to Unit in sample documentation
[Issuehunt badges](https://issuehunt.io/r/zerobias/effector/issues/140)

Link to `Unit` page in the documentation leads to nowhere and has been added by mistake, we need to remove it

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $6.00)

- [zerobias zerobias](https://issuehunt.io/u/zerobias) ($1.00)
- [issuehunt issuehunt](https://issuehunt.io/u/issuehunt) ($5.00)
### Submitted pull Requests
- [#150 [docs] Remove broken links](https://issuehunt.io/r/zerobias/effector/pull/150)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/zerobias/effector/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### PR DESCRIPTION
**Pull request by satyarohith**
This removes broken links of `Unit`.

closes #140

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#140: Remove broken link to Unit in sample documentation](https://issuehunt.io/repos/123197392/issues/140)
---

IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### CODE REVIEW

File: docs/api/effector/sample.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ hide_title: true
 ---
 
 # `sample(sourceStore, clockEvent, fn)`
-Overall this method can be used in order to link two nodes, resulting the third one, which will fire only upon `clock` node trigger.  
-_Note: [_`Unit`_](Unit.md) chapter may come in handy, to be aware of way of using this method._
+Overall this method can be used in order to link two nodes, resulting the third one, which will fire only upon `clock` node trigger.
+_Note: _`Unit`_ chapter may come in handy, to be aware of way of using this method._
```
Comment (goodmind) [inline docs/api/effector/sample.md]: This note doesn't make sense

File: docs/api/effector/Store.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ A state is reset when _Event_ or _Effect_ is called or another _Store_ is change
 
 #### Arguments
 
-- ([_`Unit`_](Unit.md)): [_`Event`_](Event.md), [_`Effect`_](Effect.md), _`Store`_
+- (_`Unit`_): [_`Event`_](Event.md), [_`Effect`_](Effect.md), _`Store`_
```
Comment (goodmind) [inline docs/api/effector/Store.md]: Change `Unit` to `Event | Effect | Store`

File: docs/api/effector/Store.md [hunk 3]
```diff
@@ -105,7 +105,7 @@ Updates state when `trigger` is triggered by using `hander`.
...
```
Comment (goodmind) [inline docs/api/effector/Store.md]: ditto

File: docs/api/effector/sample.md (hunk 1 above)
Comment (satyarohith) [reply docs/api/effector/sample.md]: Sorry about that!

**Pull request merged**